tests/features/async-components.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { defineAsyncComponent, defineComponent, h, AppConfig } from 'vue'
 
 import { mount, flushPromises } from '../../src'
+import { ComponentPublicInstance } from '@vue/runtime-core'
```

Comment:
not exported from `'vue'`? Otherwise we need to add `@vue/runtime-core` as a peer dependency.

Reply:
indeed it is, Just my IDE defaulted to this,
`vue` has: `export * from "@vue/runtime-dom";`
fixed